selfdrive/controls/controlsd.py

```diff
@@ -459,14 +469,16 @@ def state_transition(self, CS):
         else:
           if self.events.any(ET.PRE_ENABLE):
             self.state = State.preEnabled
+          elif self.events.any(ET.OVERRIDE):
+            self.state = State.overriding
```

Comment:
```suggestion
          if self.events.any(ET.PRE_ENABLE) and self.disengage_on_gas:
            self.state = State.preEnabled
          elif self.events.any(ET.OVERRIDE) and not self.disengage_on_gas:
            self.state = State.overriding
```

Can also do this and still only have one gasPressed event, but I don't know about assuming `preEnable` and `override` are always related to gas press.